RecoParticleFlow/PFClusterProducer/python/particleFlowClusterECAL_cfi.py

```diff
@@ -4,20 +4,39 @@
 
 #energy corrector for corrected cluster producer
 _emEnergyCorrector = cms.PSet(
-    algoName = cms.string("PFClusterEMEnergyCorrector"),
+    algoName = cms.string('PFClusterEMEnergyCorrector'),
     applyCrackCorrections = cms.bool(False),
     applyMVACorrections = cms.bool(True),
     maxPtForMVAEvaluation = cms.double(90.),
     recHitsEBLabel = cms.InputTag('ecalRecHit', 'EcalRecHitsEB'),
     recHitsEELabel = cms.InputTag('ecalRecHit', 'EcalRecHitsEE'),
+    ebSrFlagLabel = cms.InputTag('ecalDigis'),
+    eeSrFlagLabel = cms.InputTag('ecalDigis'),
     autoDetectBunchSpacing = cms.bool(True),
+    srfAwareCorrection = cms.bool(False)
+)
+
+_emEnergyCorrector_2017 = cms.PSet(
+    algoName = cms.string('PFClusterEMEnergyCorrector'),
+    applyCrackCorrections = cms.bool(False),
+    applyMVACorrections = cms.bool(True),
+    maxPtForMVAEvaluation = cms.double(300.),
+    recHitsEBLabel = cms.InputTag('ecalRecHit', 'EcalRecHitsEB'),
+    recHitsEELabel = cms.InputTag('ecalRecHit', 'EcalRecHitsEE'),
+    ebSrFlagLabel = cms.InputTag('ecalDigis'),
+    eeSrFlagLabel = cms.InputTag('ecalDigis'),
+    autoDetectBunchSpacing = cms.bool(True),
+    srfAwareCorrection = cms.bool(True)
 )
 
 particleFlowClusterECAL = cms.EDProducer(
-    "CorrectedECALPFClusterProducer",
-    inputECAL = cms.InputTag("particleFlowClusterECALUncorrected"),
-    inputPS = cms.InputTag("particleFlowClusterPS"),
+    'CorrectedECALPFClusterProducer',
+    inputECAL = cms.InputTag('particleFlowClusterECALUncorrected'),
+    inputPS = cms.InputTag('particleFlowClusterPS'),
     minimumPSEnergy = cms.double(0.0),
     energyCorrector = _emEnergyCorrector
     )
 
+
```

Comment:
multiple copy-paste in this file combined with well defined CorrectedECALPFClusterProducer::fillDescriptions is multiply-redundant and error-prone to maintain.
Please remove this particleFlowClusterECAL_cfi.py file or rename to particleFlowClusterECAL_cff.py with the following content, which has modifications from the defaults already set in the fillDescriptions where needed without redundant repetitions:

```python
import FWCore.ParameterSet.Config as cms

from RecoParticleFlow.PFClusterProducer.particleFlowClusterECAL_cfi import particleFlowClusterECAL
particleFlowClusterECAL.energyCorrector.applyMVACorrections = True
particleFlowClusterECAL.energyCorrector.maxPtForMVAEvaluation = 90

from Configuration.Eras.Modifier_run2_ECAL_2017_cff import run2_ECAL_2017
run2_ECAL_2017.toModify(particleFlowClusterECAL,
                        energyCorrector = dict(srfAwareCorrection = True, maxPtForMVAEvaluation = 300))
```

BTW, comparing to what I see in the HLT config, it looks like the old non-MVA corrector is used there. Is this intentional?